beacon_node/beacon_chain/src/fetch_blobs/fetch_blobs_beacon_adapter.rs

```diff
@@ -75,12 +79,28 @@ impl<T: BeaconChainTypes> FetchBlobsBeaconAdapter<T> {
         GossipVerifiedBlob::<T, DoNotObserve>::new(blob.clone(), blob.index, &self.chain)
     }
 
-    pub(crate) fn verify_data_column_for_gossip(
+    pub(crate) fn verify_data_columns_kzg(
+        &self,
+        data_columns: Vec<Arc<DataColumnSidecar<T::EthSpec>>>,
+    ) -> Result<Vec<KzgVerifiedDataColumn<T::EthSpec>>, KzgErrors> {
+        KzgVerifiedDataColumn::from_batch(data_columns, &self.chain.kzg)
```

Comment:
`KzgVerifiedDataColumn::from_batch` actually does up to two round of verifications, if batch verification fails, it attempts individual verification to identify bad columns - the purpose is to identify which peer sent the invalid columns. 

In our case here, we don't really need to know exactly which columns fails, so we can just run a single pass with `verify_kzg_for_data_column_list`